2022-02-24 15:20:54,961 INFO    MainThread:28941 [wandb_setup.py:_flush():69] setting env: {'project': 'telco_churn', 'run_group': 'development'}
2022-02-24 15:20:54,962 WARNING MainThread:28941 [wandb_setup.py:_flush():69] could not save program above cwd: /Users/jelaleddin/anaconda3/envs/mlflow-d1fd4b13aac386abe6aa5a2ada1ca11214902ce4/bin/pytest
2022-02-24 15:20:54,962 INFO    MainThread:28941 [wandb_setup.py:_flush():69] setting login settings: {}
2022-02-24 15:20:54,962 INFO    MainThread:28941 [wandb_init.py:_log_setup():334] Logging user logs to /Users/jelaleddin/MLPipeline-TelcoCustomerChurn/components/check_data/wandb/run-20220224_152054-376tdk7x/logs/debug.log
2022-02-24 15:20:54,963 INFO    MainThread:28941 [wandb_init.py:_log_setup():335] Logging internal logs to /Users/jelaleddin/MLPipeline-TelcoCustomerChurn/components/check_data/wandb/run-20220224_152054-376tdk7x/logs/debug-internal.log
2022-02-24 15:20:54,963 INFO    MainThread:28941 [wandb_init.py:init():367] calling init triggers
2022-02-24 15:20:54,963 INFO    MainThread:28941 [wandb_init.py:init():372] wandb.init called with sweep_config: {}
config: {}
2022-02-24 15:20:54,964 INFO    MainThread:28941 [wandb_init.py:init():416] starting backend
2022-02-24 15:20:54,964 INFO    MainThread:28941 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-02-24 15:20:54,985 INFO    MainThread:28941 [backend.py:ensure_launched():123] starting backend process...
2022-02-24 15:20:54,996 INFO    MainThread:28941 [backend.py:ensure_launched():127] started backend process with pid: 28948
2022-02-24 15:20:54,998 INFO    MainThread:28941 [wandb_init.py:init():421] backend started and connected
2022-02-24 15:20:55,004 INFO    MainThread:28941 [wandb_init.py:init():461] updated telemetry
2022-02-24 15:20:55,004 INFO    MainThread:28941 [wandb_init.py:init():480] communicating current version
2022-02-24 15:20:56,915 INFO    MainThread:28941 [wandb_init.py:init():485] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-24 15:20:56,916 INFO    MainThread:28941 [wandb_init.py:init():493] communicating run to backend with 30 second timeout
2022-02-24 15:20:59,017 INFO    MainThread:28941 [wandb_init.py:init():513] run resumed
2022-02-24 15:20:59,021 INFO    MainThread:28941 [wandb_init.py:init():518] starting run threads in backend
2022-02-24 15:20:59,072 INFO    MainThread:28941 [wandb_run.py:_console_start():1476] atexit reg
2022-02-24 15:20:59,073 INFO    MainThread:28941 [wandb_run.py:_redirect():1339] redirect: SettingsConsole.REDIRECT
2022-02-24 15:20:59,074 INFO    MainThread:28941 [wandb_run.py:_redirect():1344] Redirecting console.
2022-02-24 15:20:59,074 INFO    MainThread:28941 [redirect.py:install():213] install start
2022-02-24 15:20:59,074 INFO    MainThread:28941 [redirect.py:install():228] install stop
2022-02-24 15:20:59,074 INFO    MainThread:28941 [redirect.py:install():213] install start
2022-02-24 15:20:59,075 INFO    MainThread:28941 [redirect.py:install():228] install stop
2022-02-24 15:20:59,075 INFO    MainThread:28941 [wandb_run.py:_redirect():1390] Redirects installed.
2022-02-24 15:20:59,075 INFO    MainThread:28941 [wandb_init.py:init():542] run started, returning control to user process
2022-02-24 15:21:02,427 INFO    MainThread:28941 [wandb_run.py:_atexit_cleanup():1446] got exitcode: 0
2022-02-24 15:21:02,427 INFO    MainThread:28941 [wandb_run.py:_restore():1418] restore
2022-02-24 15:21:02,427 INFO    MainThread:28941 [redirect.py:uninstall():232] uninstall start
2022-02-24 15:21:02,427 INFO    MainThread:28941 [redirect.py:_stop():287] _stop: stdout
2022-02-24 15:21:02,427 INFO    MainThread:28941 [redirect.py:_stop():293] _stop closed: stdout
2022-02-24 15:21:02,428 INFO    stdout    :28941 [redirect.py:_pipe_relay():129] relay done saw last write: stdout
2022-02-24 15:21:02,428 INFO    stdout    :28941 [redirect.py:_pipe_relay():145] relay done done: stdout
2022-02-24 15:21:02,428 INFO    MainThread:28941 [redirect.py:_stop():299] _stop joined: stdout
2022-02-24 15:21:02,428 INFO    MainThread:28941 [redirect.py:_stop():301] _stop rd closed: stdout
2022-02-24 15:21:02,428 INFO    MainThread:28941 [redirect.py:uninstall():236] uninstall done
2022-02-24 15:21:02,428 INFO    MainThread:28941 [redirect.py:uninstall():232] uninstall start
2022-02-24 15:21:02,428 INFO    MainThread:28941 [redirect.py:_stop():287] _stop: stderr
2022-02-24 15:21:02,428 INFO    MainThread:28941 [redirect.py:_stop():293] _stop closed: stderr
2022-02-24 15:21:02,428 INFO    stderr    :28941 [redirect.py:_pipe_relay():129] relay done saw last write: stderr
2022-02-24 15:21:02,428 INFO    stderr    :28941 [redirect.py:_pipe_relay():145] relay done done: stderr
2022-02-24 15:21:02,428 INFO    MainThread:28941 [redirect.py:_stop():299] _stop joined: stderr
2022-02-24 15:21:02,428 INFO    MainThread:28941 [redirect.py:_stop():301] _stop rd closed: stderr
2022-02-24 15:21:02,428 INFO    MainThread:28941 [redirect.py:uninstall():236] uninstall done
2022-02-24 15:21:03,245 INFO    MainThread:28941 [wandb_run.py:_wait_for_finish():1568] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 900
}

2022-02-24 15:21:05,248 INFO    MainThread:28941 [wandb_run.py:_wait_for_finish():1568] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 1738
}

2022-02-24 15:21:07,254 INFO    MainThread:28941 [wandb_run.py:_wait_for_finish():1568] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1738
  total_bytes: 1738
}

2022-02-24 15:21:08,383 INFO    MainThread:28941 [wandb_run.py:_show_files():1790] logging synced files
